testing/py-attrs/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: Adam Dobrawy <a.dobrawy@hyperone.com>
+# Maintainer: Adam Dobrawy <a.dobrawy@hyperone.com>
+pkgname=py-attrs
+_pkgname=attrs
+pkgver=19.1.0
+pkgrel=0
+pkgdesc="Classes Without Boilerplate"
+url="https://www.attrs.org/"
+arch="noarch"
+license="MIT"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build || return 1
+	python3 setup.py build || return 1
+}
```

Comment:
No need to cd into builddir, abuild does that automatically.
No need to return 1